src/SFA.DAS.EmployerAccounts.Web/Views/Home/Privacy.cshtml

```diff
@@ -312,14 +312,14 @@
         <p>
             Most services we link to are run by different government departments, eg HM Revenue and Customs (<abbr title="Her Majesty’s Revenue and Customs">HMRC</abbr>) or the Home Office. These services may set additional cookies and, if so, will have their own cookies policy.
         </p>
-
+        @*
```

Comment:
Same as previous comment regarding commenting out this code

Reply:
Done. Replaced link to the new survey and updated text to match